LocationServices/LocationServices/Views/MapAnnotations/GeofenceAnnotation.swift

```diff
@@ -101,6 +101,12 @@ class GeofenceAnnotationView: MGLAnnotationView {
         addSubview(uiImageView)
         
         uiImageView.image = .geofenceDashoard
+        
+        uiImageView.layer.shadowColor = UIColor.black.cgColor
```

Comment:
let's create an extension on UIView to add a shadow there and we can reuse it with just passing params

Also we can have a struct Style/Settings that will collect all the needed fields in order not to pass too much params into the function. This structure also could have default values for easier configuration and also static properties for reusing configurations